inc/saiport.h

```diff
@@ -1293,7 +1312,14 @@ typedef enum _sai_port_attr_t
      * @default internal
      */
     SAI_PORT_ATTR_PORT_SERDES_ID,
-
+    
+    /** @brief Link training failure status and error codes
+     * @type sai_port_link_training_fail_status_t
+     * @flags READ_ONLY
+     * @default internal     
```

Comment:
default is not required if attribute is read only

Reply:
done